src/shadowbox/server/manager_metrics.ts

```diff
@@ -40,4 +72,71 @@ export class PrometheusManagerMetrics implements ManagerMetrics {
     }
     return {bytesTransferredByUserId: usage};
   }
+
+  async getServerMetrics({hours}: ServerMetricsTimeframe): Promise<ServerMetrics> {
```

Comment:
Let's give a name to the timeframe, so it's clearer what it is. `hours` is not that informative.

```suggestion
  async getServerMetrics(timeframe: Duration): Promise<ServerMetrics> {
```